Assets/Scripts/EnemySystem/ScriptableObjects/EnemyMovementStrategies/BabeMovementStrategy.cs

```diff
@@ -0,0 +1,188 @@
+            //Unity caches the values of Strategies variables
+            //This is to make sure the variables are reset each time that you run the game in Unity
+            #if UNITY_EDITOR
+                ResetValuesAtStart();
+            #endif
+            
+            if (playerTransform != null && emuTransform.gameObject.GetComponent<BossController>().battleStarted())
+            {
+                if (startMoveTime < Time.time)
+                {
+                    //If the player is on a platform, make Babe jump
+                    if (PlayerIsOnPlatformAbove(emuTransform, playerTransform) && !chargeAttack || jumpAttack) 
+                    {
+                        JumpToPlayer(emuTransform, playerTransform);
+                    }
+                    //Otherwise, make Babe charge
+                    else if (!jumpAttack)
+                    {
+                        ChargeTowardsPlayer(emuTransform, playerTransform);
+                    }
+                }
+                
+                //If Babe is too high in the sky, bring it back
+                BringBabeDown(emuTransform);
+                
+                //If the timestamp of the new attack is defined, set the timestamp to end it
+                if (endMoveTime <= startMoveTime)
+                    endMoveTime = startMoveTime + attackTime;
+                else if (endMoveTime <= Time.time)
+                {
+                    //Reset everything at the end of the attack
+                    chargeAttack = false;
+                    jumpAttack = false;
+                    chargeJumped = false;
+                    hasPassedPlayer = false;
+                    startMoveTime = Time.time + waitTime;
+                    emuTransform.gameObject.GetComponent<Animator>().SetBool("IsMoving", false);
+                    emuTransform.GetComponent<BoxCollider2D>().enabled = true;
+                    emuTransform.GetComponent<Rigidbody2D>().gravityScale = regularGravityScale;
+                }
+            }
+            return (chargeAttack || jumpAttack);
+        }
+
+        private void JumpToPlayer(Transform emuTransform, Transform playerTransform)
+        {
+                //If the Jump attack just started, set where Babe should jump
+                if (!jumpAttack)
+                {
+                    whereToJump = new Vector2(playerTransform.position.x, emuTransform.position.y);
+                    jumpAttack = true;
+                }
+                
+                //Move Babe to whereToJump coordinates if need to
+                if (whereToJump != Vector2.zero)
+                {
+                    emuTransform.gameObject.GetComponent<Animator>().SetBool("IsMoving", true);
+                    emuTransform.position =
+                        Vector2.MoveTowards(emuTransform.position, whereToJump, chargeSpeed * jumpSpeedScale * Time.deltaTime);
+                }
+
+                //When Babe reach her destination, make her jump
```

Comment:
i thought babe was a guy... i guess the GDD will have to change 😂

Reply:
Oh I tried to make my comment no gender, looks like I forgot to change that one oops

Reply:
is babe gender non-binary now?

Reply:
looks like it